components/mpas-ocean/src/shared/mpas_ocn_thick_ale.F

```diff
@@ -188,15 +189,16 @@ subroutine ocn_ALE_thickness(verticalMeshPool, SSH, ALE_thickness, &
                             + vertCoordMovementWeights(k) &
                             * restingThickness(k,iCell)
             end do
+            columnThickness = bottomDepth(iCell) + SSH(iCell)
    
             ! Note that restingThickness is nonzero, and remaining
             ! terms are perturbations about zero.
             ! This is equation 4 and 6 in Petersen et al 2015,
             ! but with eqn 6
             do k = kMin, kMax
-               ALE_thickness(k,iCell) = restingThickness(k,iCell) &
-                  + (SSH(iCell)*vertCoordMovementWeights(k)* &
-                     restingThickness(k,iCell) )/thicknessSum
+               ALE_thickness(k, iCell) = restingThickness(k, iCell) &
+                  + ( (columnThickness - thicknessSum) * vertCoordMovementWeights(k) * &
```

Comment:
I think this would be more clear if thicknessSum were named restingThicknessSum.